src/components/AntdPicker.js

```diff
@@ -229,15 +161,13 @@ const AntdPicker = props => {
   ]);
   const { theme, type, language, locale } = props;
 
-  useEffect(() => {
-    moment.locale(language);
```

Comment:
Import of moment locales makes the same as this? It might not change language on runtime if we delete this useEffect?

Reply:
AntdPicker is wrapped by DatePicker component, and we already have a useEffect for moment.locale on DatePicker (lines 161-163), so I have removed it.
If only the component could be used without the wrapper, it would make sense to have a useEffect in each of them. But as it is right now, the useEffect will be triggered on DatePicker and, as the "language" is a prop of AntdPicker, it will get rerender.